spec/nameable/latin_spec.rb

```diff
@@ -1,284 +1,286 @@
-end
 
-context('with an all lowercase name') do
-  subject(:n) { Nameable::Latin.new.parse("chris horn") }
-  it '.first' do
-    expect(n.first).to eq('Chris')
-  end
-  it '.last' do
-    expect(n.last).to eq('Horn')
+  context('with an all lowercase name') do
+    subject(:n) { Nameable::Latin.new.parse("chris horn") }
+    it '.first' do
+      expect(n.first).to eq('Chris')
+    end
+    it '.last' do
+      expect(n.last).to eq('Horn')
+    end
   end
-end
 
-context('with a mixed case name') do
-  subject(:n) { Nameable::Latin.new.parse("DeChris HORN") }
-  it '.first' do
-    expect(n.first).to eq('DeChris')
+  context('with a mixed case name') do
+    subject(:n) { Nameable::Latin.new.parse("DeChris HORN") }
+    it '.first' do
+      expect(n.first).to eq('DeChris')
+    end
   end
-end
 
-context('with a simple middle name') do
-  subject(:n) { Nameable::Latin.new.parse("Chris Derp Horn") }
-  it '.prefix' do
-    expect(n.prefix).to be_nil
-  end
-  it '.first' do
-    expect(n.first).to eq('Chris')
-  end
-  it '.middle' do
-    expect(n.middle).to eq('Derp')
-  end
-  it '.last' do
-    expect(n.last).to eq('Horn')
-  end
-  it '.suffix' do
-    expect(n.suffix).to be_nil
+  context('with a simple middle name') do
+    subject(:n) { Nameable::Latin.new.parse("Chris Derp Horn") }
+    it '.prefix' do
+      expect(n.prefix).to be_nil
+    end
+    it '.first' do
+      expect(n.first).to eq('Chris')
+    end
+    it '.middle' do
+      expect(n.middle).to eq('Derp')
+    end
+    it '.last' do
+      expect(n.last).to eq('Horn')
+    end
+    it '.suffix' do
+      expect(n.suffix).to be_nil
+    end
   end
-end
 
-context('with a simple prefix') do
-  subject(:n) { Nameable::Latin.new.parse("Sir Chris Horn") }
-  it '.prefix' do
-    expect(n.prefix).to eq('Sir')
-  end
-  it '.first' do
-    expect(n.first).to eq('Chris')
-  end
-  it '.middle' do
-    expect(n.middle).to be_nil
-  end
-  it '.last' do
-    expect(n.last).to eq('Horn')
-  end
-  it '.suffix' do
-    expect(n.suffix).to be_nil
+  context('with a simple prefix') do
+    subject(:n) { Nameable::Latin.new.parse("Sir Chris Horn") }
+    it '.prefix' do
+      expect(n.prefix).to eq('Sir')
+    end
+    it '.first' do
+      expect(n.first).to eq('Chris')
+    end
+    it '.middle' do
+      expect(n.middle).to be_nil
+    end
+    it '.last' do
+      expect(n.last).to eq('Horn')
+    end
+    it '.suffix' do
+      expect(n.suffix).to be_nil
+    end
   end
-end
 
-context('with a simple suffix') do
-  subject(:n) { Nameable::Latin.new.parse("Chris Horn PhD") }
-  it '.prefix' do
-    expect(n.prefix).to be_nil
-  end
-  it '.first' do
-    expect(n.first).to eq('Chris')
-  end
-  it '.middle' do
-    expect(n.middle).to be_nil
-  end
-  it '.last' do
-    expect(n.last).to eq('Horn')
-  end
-  it '.suffix' do
-    expect(n.suffix).to eq('Ph.D.')
+  context('with a simple suffix') do
+    subject(:n) { Nameable::Latin.new.parse("Chris Horn PhD") }
+    it '.prefix' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.